default.config.yml

```diff
@@ -0,0 +1,75 @@
+---
+# Role: uninstalls
+# set home directory
+home_directory: /Users/
```

Comment:
I see this is only used once. And it seems to refer to home directory for all users. So maybe it should be renamed `users_home_directory` to avoid confusion with the user's home directory?